services/movie.py

```diff
@@ -15,13 +17,17 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__contains=title)
```

Comment:
The `filter` method is using `title__contains=title`, which implies `title` should be a string. If `title` is intended to be a list of strings, you should iterate over the list and apply the filter for each title.